cli/js/web/console.ts

```diff
@@ -524,10 +558,21 @@ function createRawObjectString(
   const entries: string[] = [];
   const stringKeys = Object.keys(value);
   const symbolKeys = Object.getOwnPropertySymbols(value);
```

Comment:
A refactoring opportunity I see here, just suggestion:
```ts
cons keys = [
    ...Object.keys(value),
    ...Object.getOwnPropertySymbols(value).map(key => key.toString())
]
```
Such that the for loops that comes after don't have to be repeated twice.

Reply:
There are differences in the loop that are better to keep for explicitness.